types/yup/index.d.ts

```diff
@@ -89,6 +89,9 @@ export interface NumberSchemaConstructor {
 export interface NumberSchema extends Schema {
     min(limit: number | Ref, message?: string): NumberSchema;
     max(limit: number | Ref, message?: string): NumberSchema;
+    less(limit: number | Ref, message?: string): NumberSchema;
+    more(limit: number | Ref, message?: string): NumberSchema;
+    notEqual(limit: number | Ref, message?: string): NumberSchema;
```

Comment:
Regarding this addition, I couldn't find `notEqual` inside the [yup/number.js](https://github.com/jquense/yup/blob/v0.24.0/src/number.js). Looks like this line should be deleted.

Reply:
Dammit. Overlooked that... -.-

Reply:
You are correct, and this commit also renamed `less` and `more`…
I will update the PR, and submit one in Yup repo to fix the Changelog.

Reply:
Done, with a doc PR on Yup: https://github.com/jquense/yup/pull/167